store/src/main/java/org/apache/rocketmq/store/stats/BrokerStats.java

```diff
@@ -88,6 +88,6 @@ public long getMsgPutTotalTodayNow() {
     }
 
     public long getMsgGetTotalTodayNow() {
```

Comment:
How about dealing with getMsgGetTotalYesterdayMorning in the same way?

Reply:
The getMsgGetTotalYesterdayMorning's data is from getGetMessageTransferredMsgCount,
so it's ok to replace the getGetMessageTransferredMsgCount to getNumsWithoutSystemTopic without adding other  counter